example/src/main/assets/mobile-services.json

```diff
@@ -26,6 +26,12 @@
             "config": {
             }
         }
+    ],
+    "https": [
+        {
+            "host": "security.feedhenry.org",
+            "certificateHash": "292B039CAC326C7A81450C7C21F4235C6568DCF9F2A74FBB7E977318462DCABD="
```

Comment:
This needs to be a Base64 encoded Subject Public Key Information fingerprint to work with OkHTTP's pinner. The mobile-cli should output `trENjoQnbWupnAtu1/WagBE0RgJ+p7ke2ppWML8vAl0=` for `security.feedhenry.org`.

Reply:
@TommyJ1994 The actual output of the Mobile CLI for `security.feedhenry.org` is.

```json
...
"https": [
    {
        "host": "security.feedhenry.org",
        "certificateHash": "trENjoQnbWupnAtu1/WagBE0RgJ+p7ke2ppWML8vAl0="
    }
]
...
```

so the CLI _should_ not cause any issues here when we're actually using it.